app/models/incident.rb

```diff
@@ -51,8 +59,8 @@ class Incident < ApplicationRecord
   after_create :send_notifications
 
   def claim_for(user)
-    supervisor_incident_report = create_supervisor_incident_report user: user
-    supervisor_report = create_supervisor_report
+    self.supervisor_incident_report = create_supervisor_incident_report user: user
```

Comment:
just curious, why'd you have to do this?

Reply:
Resolved a warning about setting an unused variable. I didn't want it to come across as such, so I thought I'd make it explicit with a `self`.

Didn't have to, just thought it made clearer what I was doing.